DOCS/openfga-how-to.md

```diff
@@ -0,0 +1,214 @@
+This document shows how to set up a fully working OpenFGA server using our charm, MicroK8s and Juju 
+
+OpenFGA requires a way to persist data, in the case of our charm we enforce the usage of a `postgreSQL` database
+
+
+## Juju setup
+
+
+Bootstrap a [microk8s controller](https://juju.is/docs/olm/microk8s) using juju `3.2` and create a new Juju model:
```

Comment:
Perhaps it would be better to point to https://juju.is/docs/juju/set-up--tear-down-your-test-environment#heading--set-up-automatically?